CMakeLists.txt

```diff
@@ -156,20 +157,49 @@ set_source_files_properties("thirdparty/loguru/loguru.cpp"
 find_package(Threads REQUIRED)
 
 # create the lslboost target
-add_library(lslboost OBJECT
-	lslboost/serialization_objects.cpp
-)
-target_link_libraries(lslboost PUBLIC Threads::Threads)
-target_compile_features(lslboost PUBLIC cxx_std_11 cxx_lambda_init_captures)
-
-target_compile_definitions(lslboost
-	PUBLIC
-		BOOST_ALL_NO_LIB
-		BOOST_ASIO_STANDALONE
-		BOOST_THREAD_DONT_PROVIDE_INTERRUPTIONS
-)
-target_include_directories(lslboost SYSTEM PUBLIC
-	${CMAKE_CURRENT_SOURCE_DIR}/lslboost)
+if(LSL_BUNDLED_BOOST)
+	message(STATUS "Using bundled Boost")
+		
+	add_library(lslboost OBJECT
+		lslboost/serialization_objects.cpp
+	)
+	target_link_libraries(lslboost PUBLIC Threads::Threads)
+	target_compile_features(lslboost PUBLIC cxx_std_11 cxx_lambda_init_captures)
+
+	target_compile_definitions(lslboost
+		PUBLIC
+			BOOST_ALL_NO_LIB
+			BOOST_ASIO_STANDALONE
+			BOOST_THREAD_DONT_PROVIDE_INTERRUPTIONS
+	)
+	target_include_directories(lslboost SYSTEM PUBLIC
+		${CMAKE_CURRENT_SOURCE_DIR}/lslboost)
+else()
+	message(STATUS "Using system Boost")
+
+	find_package(Boost REQUIRED
+		COMPONENTS
+			serialization
+	)
+	add_library(lslboost INTERFACE)
+	target_compile_definitions(lslboost
+		INTERFACE
+			BOOST_ALL_NO_LIB
+			BOOST_ARCHIVE_SOURCE
+			BOOST_ASIO_STANDALONE
+			#BOOST_ASIO_SEPARATE_COMPILATION
+			BOOST_THREAD_DONT_PROVIDE_INTERRUPTIONS
+			$<$<PLATFORM_ID:Windows>:_WIN32_WINNT=${LSL_WINVER}>
+			lslboost=boost # allows the LSL code base to work with the original Boost namespace/headers
+	)
+	target_link_libraries(lslboost
```

Comment:
This is already linked in in line 201

Reply:
Removed in b9d2b96b57feda4f244f2b6002e8225497a4b337